authors/apps/profiles/models.py

```diff
@@ -0,0 +1,45 @@
+
+from django.db import models
+from ..authentication.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class Profile(models.Model):
+    """
+    create a user profile model
+    """
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    username = models.CharField(max_length=20, blank=False)
+    bio = models.TextField(blank=True)
+    image = models.URLField(
+        blank=True,
+        default="https://d1nhio0ox7pgb.cloudfront.net/_img/o_collection_png/green_dark_grey/512x512/plain/user.png"
+    )
+    company = models.CharField(max_length=100, blank=True)
```

Comment:
Regarding the field that have been added, I believe we should consult with the PO and the TTL to clarify what is required.

Reply:
With regard to the following field, I believe that was your task. And about the fields thats where creativity came in as described in the task. But I agree with you on consulting with the PO and the TTL to clarify what is required.